CHANGELOG.md

```diff
@@ -39,7 +39,10 @@ All notable changes to this project will be documented in this file.
 * ### NI-SCOPE
     * #### Added
     * #### Changed
-        * `Session.fetch()` and `Session.fetch_into()` now takes additional parameters for associated attributes and add resulting record as part of the waveform info
+        * `Session.fetch()` and `Session.fetch_into()` updated
+            * now takes additional parameters for associated attributes 
+            * add resulting record as part of the waveform info
```

Comment:
I never know how much detail you are expecting in the changelog. Other times when I have put that type of information you said it belonged in the documentation and not the changelog. So that was what I did. I made sure that level of detail is in the documentation.

Added links to documentation instead.